web/views.py

```diff
@@ -44,3 +44,6 @@ def logout(request):
     auth_logout(request)
     return redirect('/')
 
+def member_page(request):
+	user = UserProfile.objects.get(githubuser="PettraV")
```

Comment:
Like I commented above for the url: use argument in view to filter your data.
... and  please cancel the hardcoded user... :weary: